F21/nikhil3799/main.py

```diff
@@ -0,0 +1,41 @@
+class Word:
+    def __init__(self, word, code):
+        self.word = word
+        self.code = code
+
+    def low_er(self):

```

Comment:
Use better naming convention for functions as it is difficult to understand what this function is for, 
instead of "low_er" you can use naming convention like "lowerCaseConverter" or "functionLowerCase", etc.